revised test prep title; added NGSS tags; title documentation

kjdav opened the pull request:
- Revised the title tag for Test Prep from Multiple Choice to Test Prep Multiple Choice. We need to include Test Prep in the title tag for all Test Prep question (multiple choice, short answer, extended response). 
- W&N will add NGSS (Next Generation Science Standards) tags to chapter-review-performance items. These standards are defined externally to Tutor, and teachers will see them in TS content. To make sure they can be treated similarly to LO's, I added
  ost-tag-ngss-k12phys-\* for NGSS items and ost-ngss-def class on the corresponding TS (see changes to Conventions, Performance Task example, and Teacher Content with NGSS example)
- Added documentation to explain Chapter Review and Test Prep <title> tags

Commit:
revised test prep title; added NGSS tags; title documentation

- Revised the <title> tag for Test Prep from Multiple Choice to Test Prep Multiple Choice. We need to include Test Prep in the <title> tag for all Test Prep question (multiple choice, short answer, extended response). 

- W&N will add NGSS (Next Generation Science Standards) tags to chapter-review-performance items. These standards are defined externally to Tutor, and teachers will see them in TS content. To make sure they can be treated similarly to LO's, I added
ost-tag-ngss-k12phys-* for NGSS items and ost-ngss-def class on the corresponding TS (see changes to Conventions, Performance Task example, and Teacher Content with NGSS example)

- Added documentation to explain Chapter Review and Test Prep <title> tags

Commit:
ngss tag on text instead of ngss number

Alina pointed out we should have the NGSS span tag around the text of the NGSS, rather than the identifier. Changed.

Commit:
Demonstration label is now Teacher Demonstration

philschatz commented on README.md:
```diff
@@ -257,6 +259,16 @@ Notes:
+
+<note class="os-teacher">
+  <label>Teacher Edition</label>
+  <p>NGSS HS-PS2-1:<span class="ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze
```
The `<span class="ost-tag-ngss">` should probably exclude "Students who demonstrate understanding can: " since I'm guessing it'll be repeated for every tag, no?

openstaxalina replied:
@philschatz we were trying to structure it more like how the TEKS are defined; the "NGSS HS-PS2-1" seems easier for the XML teams to locate than a phrase with a colon after it.  Technically W&N's markup is inconsistent between TEKS defined and NGSS defined...I could see us going either way.  Thoughts? (@kjdav ?)

kjdav replied:
I tend towards leaving it as-is, based on the possibility of inconsistent handling by the XML team who has to locate the second colon in a paragraph. However, if Phil feels strongly, we could definitely change it in the TL and see how the XML team handles it in the first chapter they send back?

philschatz replied:
ok, It just means the UX will need to decide what the human-readable version of the tag is. Either `NGSS HS-PS2-1` or `Students who demonstrate understanding can: Analyze ...`

kjdav replied:
Phil makes a very good point. We are only targeting the text of the TEKS/NGSS standards, but we should target both the identifier and the text. I will add another commit. 

Phil suggests: `<p class=“tag-def”><span class=“tag-name”>NGSS HS-PS2-1</span><span class=“tag-split”>:</span><span class=“tag-desc ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze…</span></p>`

philschatz replied:
You need to use backticks "`" when writing elements so they don't disappear.

philschatz commented:
Just one comment (noted above). Otherwise looks great!

Commit:
important updates to ost-tag-lo-* locations + extra documentation

- added ost-tag-lo-k12phys-* class to elements that were missing it in the GH TL: all text features, snap labs, virtual physics, watch physics, worked example/s. Not sure how this was missed in GH, but all these elements have S-LO tags in the Robust Content, so should be here. 
- conventions: ost-assignable is on snap labs AND performance tasks
- visual only: changed example-problem/s to worked-example/s to match the rest of the TL

Commit:
typo fix: added `` around ost-tag-lo- where needed

Commit:
changed iframe embed codes per Phil/Alina

added: <media alt=“…” class=“os-embed><iframe width=“960” height=“785” src=“…”/></media>

Commit:
new treatment for NGSS and TEKS standards

PHIL: Spoke with Kathi and decided on this simplified version of what you suggested. Please do review :) 
- new TS content example defining NGSS standards (includes ost-standards-ngss, ost-standards-def, ost-standards-name, ost-standards-description tags)
- new TS content example defining TEKS standards (includes ost-standards-teks, ost-standards-def, ost-standards-name, ost-standards-description tags)
- added all new tags to conventions and full list of tags
- removed now extraneous tags ost-ngss-def and ost-teks-def

philschatz commented on README.md:
```diff
@@ -248,15 +253,39 @@ Notes:
+	   student is expected to:
+	   </item>
+      <list>
+          <item class="ost-standards-def ost-standards-teks"><span class="ost-standards-name">(4C)</span>: <span   
```
The `:` will also need a `<span class="ost-standards-discard">` on it so UX folks can style it too (remove it). Basically, if we want to do fine-grained styling on text in the item (even to hide it) then we need a class on it.

Dantemss replied:
Do we even need the `:`?

philschatz replied:
That's a question for the textbook folks. If they want to add special styling (`...:after { content: ": "; }`) then they'll need a non-`ost-` class name on the `ost-standards-name` element.

Commit:
added ost-standard-discard

see changes to Conventions, list of tags, and TS content with NGSS and TEKS standards
